libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -32,12 +32,13 @@
           <p [innerHTML]="book.description"></p>
           <div>
             <button
+              data-testing="add-to-reading-list"
               mat-flat-button
               color="primary"
               (click)="addBookToReadingList(book)"
-              [disabled]="book.isAdded"
+              [disabled]="book.isAdded || book.finished"
```

Comment:
Do you need to do this? Surely a book that is finished is _also_ added?

Reply:
Right. Reverted to book.isAdded check.